cmd/gitserver/server/cleanup.go

```diff
@@ -312,7 +339,14 @@ func (s *Server) freeUpSpace(howManyBytesToFree int64) error {
 
 	// Remove repos until howManyBytesToFree is met or exceeded.
 	var spaceFreed int64
+	mountPoint, err := findMountPoint(s.ReposDir)
+	if err != nil {
+		return errors.Wrap(err, "finding mount point")
+	}
```

Comment:
```suggestion
	}
	
```
nit: https://github.com/sourcegraph/sourcegraph/blob/master/doc/dev/code_style_guide.md#use-new-lines-intentionally

Reply:
I want to keep that block of code contiguous because there is a comment above it that describes what the whole thing is doing. Adding a line break would give the idea that the comment doesn't talk about some code that it is meant to talk about.

Reply:
I found another place where I can add a newline to make it more readable. Thanks for the suggestion.